external/loaders/loaders/mappers/_nudged/_nudged.py

```diff
@@ -179,10 +182,10 @@ def open_nudge_to_fine(
         "y_wind_tendency_due_to_nudging": "dQywind",
         "eastward_wind_tendency_due_to_nudging": "dQu",
         "northward_wind_tendency_due_to_nudging": "dQv",
-        "tendency_of_air_temperature_due_to_fv3_physics": "pQ1",
-        "tendency_of_specific_humidity_due_to_fv3_physics": "pQ2",
-        "tendency_of_eastward_wind_due_to_fv3_physics": "pQu",
-        "tendency_of_northward_wind_due_to_fv3_physics": "pQv",
+        f"tendency_of_air_temperature_due_to_{gsrm_name}_physics": "pQ1",
+        f"tendency_of_specific_humidity_due_to_{gsrm_name}_physics": "pQ2",
+        f"tendency_of_eastward_wind_due_to_{gsrm_name}_physics": "pQu",
+        f"tendency_of_northward_wind_due_to_{gsrm_name}_physics": "pQv",
```

Comment:
How much control do we have over the SCREAM output names? Do you know what the names of these quantities (physics tendencies) will be? 

And if they're not named `tendency_of_air_temperature_due_to_scream_physics`, probably we should just not make this change. Note that this function does not fail if these variables don't exist in the given dataset.

Reply:
Apparently, it is possible for us to change scream output names (in the future). Currently there isn't any physics tendency variables, so I'm keeping the same naming convention here when making fake data.